portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +30,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function toggle (section) {
+  const container = document.getElementById(section);
+  const other = document.getElementById(section === 'swimming' ? 'rowing': 'swimming');
```

Comment:
nit: rename to something more descriptive? While the starters are simple, in larger codebases it's good to name functions/variables clearly so someone skimming could quickly understand what the function/variables are for without needing to dig deeply into the code. 

something like - function toggleVisibility, const displayDiv, const hideDiv

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers